optimum/intel/openvino/quantization.py

```diff
@@ -360,7 +358,7 @@ def _quantize_torchmodel(
             logger.info(
                 "No configuration describing the quantization process was provided, a default OVConfig will be generated."
             )
-            ov_config = OVConfig(compression=DEFAULT_QUANTIZATION_CONFIG)
+            ov_config = OVConfig()
```

Comment:
not used for quantization (but now only for the `save_onnx_model` parameter) so not sure we need to create an instance when not provided, we can only give `save_onnx_model` a default value instead. Also no need to save the configuration after the quantization + export steps for the same reason  : would remove saving the config here 
https://github.com/huggingface/optimum-intel/blob/c2d267aef2fc2185be8cece275fc27e13f9b3602/optimum/intel/openvino/quantization.py#L444

Reply:
Made the suggested changes, also removed ov_config from list of arguments altogether. In the future will bring it back to pass quantization parameters through it.